octobot/community/authentication.py

```diff
@@ -54,7 +54,7 @@ async def bot_data_update_wrapper(*args, raise_errors=False, **kwargs):
                 raise err
             self.logger.exception(err, True, f"Error when calling {func.__name__} {err}")
         finally:
-            self.logger.debug(f"bot_data_update: {func.__name__} completed.")
+            self.logger.info(f"bot_data_update: {func.__name__} completed.")
```

Comment:
Should we really change this one?

Reply:
right, i'll change it

Reply:
up: I moved it to open orders update as those are important to sync